doc/source/release-notes/1.6.0.rst

```diff
@@ -0,0 +1,48 @@
+.. _PBM-1.6.0:
+
+================================================================================
+*Percona Backup for MongoDB* 1.6.0
+================================================================================
+
+:Date: August 5, 2021
+:Installation: `Installing Percona Backup for MongoDB <https://www.percona.com/doc/percona-backup-mongodb/installation.html>`_
+
+Percona Backup for MongoDB is a distributed, low-impact solution for consistent backups of MongoDB
+sharded clusters and replica sets. This is a tool for creating consistent backups
+across a MongoDB sharded cluster (or a single replica set), and for restoring
+those backups to a specific point in time. 
+
+Release highlights:
+
+- Support for Percona Server for MongoDB and MongoDB Community 5.0
+- Point-in-time recovery enhancements: ability to restore from any previous snapshot and configurable span of oplog events
+- JSON output for PBM commands to simplify interfacing PBM with applications
+
+
+New Features
+================================================================================
+
+* :jirabug:`PBM-617`: Ability to restore from previous snapshots to point-in-time
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PBM-543`: Configure the size of the span of oplog events for point-in-time recovery
+* :jirabug:`PBM-652`: Add a command to delete PITR chunks
+* :jirabug:`PBM-632`: Add JSON output for all commands
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PBM-403`: Mask user credentials in ``ps`` output of ``pbm-agent``
+* :jirabug:`PBM-694`: Fix restoring from a backup when it contains VIEWS collection (Thanks to Danish Qamar for reporting this issue)
+* :jirabug:`PBM-647`: Fix frequency of getting an S3 object header (Thanks to Ryan Gunner for reporting this issue)
```

Comment:
Maybe change ticket name to something like `Reduce frequency of S3 header get requests during agent health checks`. Now it's not clear where and for what reason S3 gets are happening. 
I am also not sure if it's a bug but not improvement